src/westpa/core/binning/mab.py

```diff
@@ -24,6 +24,9 @@ def map_mab(coords, mask, output, *args, **kwargs):
     if not np.any(mask):
         return output
 
+    if skip is None:
+        skip = [False] * ndim
```

Comment:
Since we're using 0 and 1s for skip, this should be `skip = [0] * ndim` for consistency

Reply:
Nice catch. Just fixed it.